### PR TITLE
Azdo updates: change log, pager response, avoid pipeline duplicate

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

**Review by hattan — CHANGES_REQUESTED** [review]
Thanks for adding this. This is a good pattern to use with the azdo api.

File: cli/azd/pkg/azdo/pipeline.go [hunk 1]
```diff
@@ -87,25 +89,15 @@ func CreatePipeline(
...
-		}
-
-		if count >= maxTries {
-			return nil, fmt.Errorf("error creating new pipeline")
-		}
+	// Add the name of the repo as part of the Pipeline name
+	name = fmt.Sprintf("%s (%s)", name, repoName)
+	exists, err := pipelineExists(ctx, client, &projectId, &name)
+	if err != nil {
+		return nil, fmt.Errorf("creating pipeline: validate name: %s", err)
```
[the remark this answers is not in the document]
Comment (vhvb1989) [reply cli/azd/pkg/azdo/pipeline.go]: done

**Review by ellismg — APPROVED** [review]
LGTM

**Pull request merged**